sufia-models/app/models/concerns/sufia/ability.rb

```diff
@@ -45,10 +46,16 @@ def stats_abilities
       alias_action :stats, to: :read
     end
 
+    def single_use_links_abilities
+      can :read, SingleUseLink 
```

Comment:
This says, every user can read every SingleUseLink.  I'm quite certain that wasn't the previous behavior.